### PR TITLE
Change directory structure to match Composer and update to release packages

### PR DESCRIPTION
**Pull request by chrimc62**
This organizes generated files to be in the way Composer expects, i.e.
language-generation\<locale>\<prop>
language-understanding\<locale>\<prop>
knowledge-base\<locale>\<prop>
dialogs\<prop>
Added scripts to build schema when projects change
Also updates to release 4.11

### CODE REVIEW

File: experimental/generation/generator/packages/library/src/mergeAssets.ts [hunk 1]
```diff
@@ -154,16 +150,14 @@ export async function mergeAssets(schemaName: string, oldPath: string, newPath:
             await fs.ensureDir(ppath.join(mergedPath, locale))
             feedback(FeedbackType.message, `Create output dir : ${mergedPath} `)
 
-            let oldFileList = []
-            getFiles(oldPath, oldFileList)
-            let newFileList = []
-            getFiles(newPath, newFileList)
+            let oldFileList = await getFiles(oldPath)
```
Comment (coldplaying42) [inline experimental/generation/generator/packages/library/src/mergeAssets.ts:153]: Should the function be getFiles(oldPath, oldFileList) ?

File: experimental/generation/generator/packages/library/templates/professional/config.json.lg [hunk 2]
```diff
@@ -11,7 +11,7 @@ ${join(foreach(locales, locale, localeTemplate(locale)), ','+EOL())}
 ```
 
 # localeTemplate(locale)
--```    "./${locale}/${prefix}.${locale}.lu": {
+-```    "./${knowledgeDir()}${locale}/${prefix}.${locale}.lu": {
```
Comment (coldplaying42) [inline experimental/generation/generator/packages/library/templates/professional/config.json.lg:14]: For the latest format for cross-train, ./${knowledgeDir()}${locale}/${prefix}.${locale}.lu should be changed to ${prefix}.${locale}

File: experimental/generation/generator/packages/library/templates/standard/luis.lu.dialog.lg [hunk 3]
```diff
@@ -1,5 +1,7 @@
+[generator.lg](../generator.lg)
+
 # filename
-- ${locale}/${prefix}.${locale}.lu.dialog
+- ${recognizerDir()}/${prefix}.${locale}.lu.dialog
```
Comment (coldplaying42) [inline experimental/generation/generator/packages/library/templates/standard/luis.lu.dialog.lg:4]: remove /

File: experimental/generation/generator/packages/library/src/mergeAssets.ts (hunk 1 above)
Comment (chrimc62) [reply experimental/generation/generator/packages/library/src/mergeAssets.ts:164]: I think this is OK.

File: experimental/generation/generator/packages/library/templates/professional/config.json.lg (hunk 2 above)
Comment (chrimc62) [reply experimental/generation/generator/packages/library/templates/professional/config.json.lg:14]: I'm not sure how this fits in with composer, so I'm leaving it right now.

File: experimental/generation/generator/packages/library/templates/standard/luis.lu.dialog.lg (hunk 3 above)
Comment (chrimc62) [reply experimental/generation/generator/packages/library/templates/standard/luis.lu.dialog.lg]: Nice catch

**Pull request merged**